sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -524,10 +537,6 @@ export class StreamingReceiver extends MessageReceiver {
    */
   private async _subscribeImpl(caller: "detach" | "subscribe"): Promise<void> {
     try {
-      // this allows external callers (ie: ServiceBusReceiver) to prevent concurrent `subscribe` calls
-      // by not starting new receiving options while this one has started.
```

Comment:
Can we retain this comment for resume()?

Reply:
Done. Re-attached it to the resume() call

Reply:
reads strange. Is this for the whole _subscribeImpl() method instead?

Reply:
actually, am not sure what part of the code here is preventing concurrent subscribe calls...

Reply:
chat with @richardpark-msft offline. We agreed that the comments can be removed